```
/Users/buildnode/jenkins/workspace/swift-PR-macos/branch-main/swift/test/Sanitizers/tsan/racy_async_let_fibonacci.swift:70:11: error: CHECK: expected string not found in input
// CHECK: ThreadSanitizer: Swift access race
          ^
```

This is definitely unrelated.

--- a/test/api-digester/Outputs/cake-abi.json
+++ b/test/api-digester/Outputs/cake-abi.json
@@ -1846,8 +1846,7 @@
           "name": "CodingKeyRepresentable",
           "printedName": "CodingKeyRepresentable",
           "usr": "s:s22CodingKeyRepresentableP",
-          "mangledName": "$ss22CodingKeyRepresentableP",
-          "isABIPlaceholder": true
+          "mangledName": "$ss22CodingKeyRepresentableP"
         },
         {
           "kind": "Conformance",

--- a/test/api-digester/Outputs/cake.json
+++ b/test/api-digester/Outputs/cake.json
@@ -1713,8 +1713,7 @@
           "name": "CodingKeyRepresentable",
           "printedName": "CodingKeyRepresentable",
           "usr": "s:s22CodingKeyRepresentableP",
-          "mangledName": "$ss22CodingKeyRepresentableP",
-          "isABIPlaceholder": true
+          "mangledName": "$ss22CodingKeyRepresentableP"
         },
         {
           "kind": "Conformance",

